_posts/2020-12-21-multipleSchemas.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: blog
+subtitle: Handling multiple schemas in the same database with Flyway
+permalink: /blog/multipleSchemas.html
+author: julia
+---
+
+We've had some users ask recently about multi-schema deployments. There are a number of possible scenarios, some
+of which are supported naturally by Flyway and some not. This blog post summarises the current state of play
+on the matter, but we're keen to make sure we cover popular use cases. If you have any feedback for us, 
+there's an ongoing discussion on [Github Issues](https://github.com/flyway/flyway/issues/2995).
+
+## Configuration
+
+There are two main configuration parameters; `flyway.schemas` and `flyway.defaultSchema`. In this post we'll talk
```

Comment:
I think this semi-colon should be a colon? Though I don't remember much from my English lessons way back when 😬